### PR TITLE
[MM-60278] app/config.go: add a nil check for a required field

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
Looks like a trivial issue due to a forgotten `nil` check.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-60278

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by agnivade — APPROVED** [review]
Nice one! Difficult to add a unit test I guess?

**Review by agarciamontoro — APPROVED** [review]

File: server/channels/app/config.go [hunk 1]
```diff
@@ -235,7 +235,7 @@ func (a *App) HandleMessageExportConfig(cfg *model.Config, appCfg *model.Config)
 	// appropriate value. The rewriting occurs here to ensure it doesn't affect values written to the config file
 	// directly and not through the System Console UI.
 	if *cfg.MessageExportSettings.EnableExport != *appCfg.MessageExportSettings.EnableExport {
-		if *cfg.MessageExportSettings.EnableExport && *cfg.MessageExportSettings.ExportFromTimestamp == int64(0) {
+		if *cfg.MessageExportSettings.EnableExport && (cfg.MessageExportSettings.ExportFromTimestamp == nil || *cfg.MessageExportSettings.ExportFromTimestamp == int64(0)) {
```
[the remark this answers is not in the document]
Comment (agnivade) [reply server/channels/app/config.go:238]: Oh nice one indeed! I saw that once and slipped my mind.

**Comment by isacikgoz** [comment]
> Nice one! Difficult to add a unit test I guess?

Not impossible, but I assume it wouldn't add too much value. I think it's just a semantic error.

**Pull request merged**